app/forms/openid_connect_authorize_form.rb

```diff
@@ -246,6 +280,7 @@ def extra_analytics_attributes
       redirect_uri: result_uri,
       scope: scope&.sort&.join(' '),
       acr_values: acr_values&.sort&.join(' '),
+      vtr: vtr,
```

Comment:
if we're adding a new analytics attribute should we update it in `analytics_events.rb` too?

Reply:
Yeah I'll put it in there.